specification/keyvault/resource-manager/readme.typescript.md

```diff
@@ -10,6 +10,92 @@ typescript:
   azure-arm: true
   generate-metadata: true
   generate-readme-md: true
+
+directive: 
```

Comment:
This directive is to keep the orignal parameter order and to mitigate the un-necessary breaking introduced by migration.